src/base/system_parameter.c

```diff
@@ -737,6 +737,8 @@ static const char sysprm_ha_conf_file_name[] = "cubrid_ha.conf";
 
 #define PRM_NAME_ORACLE_STYLE_DIVIDE "oracle_style_divide"
 
+#define PRM_NAME_MEMORY_MONITORING "memory_monitoring"
```

Comment:
How about this to say it a flag.
```suggestion
#define PRM_NAME_MEMORY_MONITORING "enable_memory_monitoring"
```

Reply:
> Is it a spec to set it NO explicitly though it's by default? If it's not, I think we should remove this.

He is right. I think it would be good to romove it.

Reply:
Done. I remove it. And I think your suggestion "enable_memory_monitoring" is better than "memory_monitoring". I change it.